account/account_test.go

```diff
@@ -440,8 +438,8 @@ func TestSignMOCK(t *testing.T) {
 		account, err := account.NewAccount(mockRpcProvider, test.Address, test.Address.String(), ks, 0)
 		require.NoError(t, err, "error returned from account.NewAccount()")
 
-		msg := utils.TestHexToFelt(t, "0x73cf79c4bfa0c7a41f473c07e1be5ac25faa7c2fdf9edcbd12c1438f40f13d8")
-		sig, err := account.Sign(context.Background(), msg)
+		// msg := utils.TestHexToFelt(t, "0x5d307ad21a407ab6e93754b2fca71dd2d3b28313f6e844a7f3ecc404263a406")
```

Comment:
We can delete this line

Reply:
Didn't find this line in the code. Hence, considered fixed.